File: src/Plane.js
// Plane.js

class Plane {
  constructor(airborn = false) {
    this.airborn = airborn;
  }

  inTheAir() {
    this.airborn = true;
  }

  landed() {
    this.airborn = false;
  }
}

export default Plane;

File: ruby/lib/plane.rb
class Plane

  attr_reader :airborne

  def initialize(airborne = false)
    @airborne = airborne
  end

  def take_off
    @airborne = true
  end

  def landed
    @airborne = false
  end

  def airborne?
    @airborne
  end
end
